spec/controllers/editorial/news_controller_spec.rb

```diff
@@ -0,0 +1,80 @@
+require 'rails_helper'
+
+RSpec.describe Editorial::NewsController, type: :controller do
```

Comment:
Yeah but its a false economy. Unless you're doing proper view specs you're not actually testing your controllers properly (and even then the relationship between views and controllers isn't necessarily tested).

Rails controllers should be considered as integration tests and they're a lot fast than full feature/acceptance tests too.